examples/communication_child_to_parent/index.html

```diff
@@ -4,8 +4,8 @@
     <meta charset="utf-8" />
     <title>Yew • Child-to-Parent Communication</title>
     <link data-trunk rel="rust" />
-    <link data-trunk rel="sass" href="index.scss" />
+    <script src="https://cdn.tailwindcss.com/3.2.6"></script>
```

Comment:
I like the separation of parent and child components into separate modules, as would be done in the real world. But I don't think this is the right place to introduce Tailwind as it bloats the HTML in what needs to be simple component examples.

Don't get me wrong, I love Tailwind, but I think it's probably a better idea to create a separate example on using Tailwind with Yew. Perhaps you could create an assortment of examples for using different css libraries 🙂